CONTRIBUTING.md

```diff
@@ -135,13 +135,13 @@ improperly referenced, and the commit is not signed off by the author.
 
 Please read the [Eclipse Foundation policy on accepting contributions via Git](http://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
 
-Your contribution cannot be accepted unless you have an [Eclipse Foundation Contributor License Agreement](http://www.eclipse.org/legal/CLA.php) in place.
+Your contribution cannot be accepted unless you have an [ECA - Eclipse Foundation Contributor Agreement](http://www.eclipse.org/legal/ECA.php) in place.
```

Comment:
yeah, that's a good point Matthew. I updated the text a bit more to take that into account.
Thanks!